src/main/kotlin/brokers/PgBroker.kt

```diff
@@ -0,0 +1,153 @@
+package brokers
+
+import MDCContext
+import com.zamna.kotask.*
+import kotlinx.coroutines.*
+import kotlinx.serialization.decodeFromString
+import kotlinx.serialization.encodeToString
+import kotlinx.serialization.json.Json
+import org.jetbrains.exposed.sql.*
+import org.jetbrains.exposed.sql.SqlExpressionBuilder.eq
+import org.jetbrains.exposed.sql.javatime.CurrentTimestamp
+import org.jetbrains.exposed.sql.javatime.timestamp
+import org.jetbrains.exposed.sql.transactions.transaction
+import plugins.scheduler.pg.DbWrapper
+import java.time.Instant
+import java.util.UUID
+import kotlin.time.Duration
+import kotlin.time.Duration.Companion.milliseconds
+import kotlin.time.Duration.Companion.minutes
+
+
+object KotaskMessages : Table(name=Settings.messagesTableName) {
+    val id = uuid("id").autoGenerate()
+    val callId = uuid("call_id")
+    val queueName: Column<String> = varchar("queue", 512)
```

Comment:
shouldn't be indexed?

Reply:
It is a part of composite index. "message_selection"